CHANGELOG.rst

```diff
@@ -4,10 +4,10 @@ Changelog
 This document describes changes between each past release.
 
 
-3.2.0 (unreleased)
+3.1.5 (2016-05-17)
 ------------------
 
-- Nothing changed yet.
+- Expose Vary: Authorization headers in every responses. (#593)
```

Comment:
I would suggest that we add it to a Bug fixes section like others :)

Also, maybe we should describe the bug itself. 
`Prevent the browser to cache server responses between two sessions` (??)